src/index.html

```diff
@@ -1,13 +1,307 @@
+                Галерея
+              </a>
+            </li>
+            <li class="menu__item">
+              <a class="menu__link" href="#events">
+                Виставки та події
+              </a>
+            </li>
+          </div>
+          <div class="menu__wrap">
+            <li class="menu__item">
+              <a class="menu__link" href="#subscription">
+                Про Нас
+              </a>
+            </li>
+            <li class="menu__item">
+              <a class="menu__link" href="#contacts">
+                Контакти
+              </a>
+            </li>
+          </div>
+
+        </ul>
+      </nav>
+
+      <div class="menu__grafic grafic">
+        <p class="grafic__title">
+          Графік роботи
+        </p>
+
+        <div class="grafic__week">
+          <p><span class="grafic__bold">12:00 — 20:00</span> середа</p>
+          <p><span class="grafic__bold">12:00 — 20:00</span> четвер</p>
+          <p><span class="grafic__bold">12:00 — 20:00</span> п’ятниця</p>
+          <p><span class="grafic__bold">11:00 — 19:00</span> субота</p>
+          <p><span class="grafic__bold">11:00 — 19:00</span> неділя</p>
+        </div>
+
+
+        <div class="grafic__kasa">
+          <p><span class="grafic__bold">Вихідні:</span> понеділок, вівторок
+            <br>Каса припиняє роботу за 1 годину до закриття музею
+          </p>
+        </div>
+      </div>
+    </div>
+  </aside>
+
+  <header class="header page__header page__container" id="header">
+    <a href="#menu" class="header__menu page__icon"></a>
+
+    <a href="#" class="header__logo">
+      <img src="./images/icons/namu-logo.svg" alt="namu logo" class="header__img">
+    </a>
+  </header>
+
+  <a href="#" class="page__wrapper"></a>
+
+  <main class="page__main">
+    <section id="museum" class="museum">
+      <div class="museum__top">
+      </div>
+
+      <div class="museum__bottom">
+        <h1 class="museum__title">ХУДОЖНІЙ МУЗЕЙ</h1>
+        <a href="#" class="tickets museum__tickets">
+          <p class="tickets__text">
+            Квитки
+          </p>
+          <div class="tickets__arrow">
+            <img src="./images/icons/arrow.svg" alt="arrow" class="tickets__image">
+          </div>
+        </a>
+      </div>
+    </section>
+
+    <section id="events" class="events page__events">
+      <h2 class="events__title page__title">Актуальні події</h2>
+      <article class="article events__article">
+        <img src="./images/images/event_1.jpg" alt="image" class="article__image">
+
+        <div class="article__wrap">
+          <div class="article__date">
+            <p>ВИСТАВКА</p>
+            <p class="article__dating">26.08-29.11.2019</p>
+          </div>
+          <a href="#" class="article__title">Йду і повертаюсь</a>
+          <p class="article__text">Національний Художній Музей України презентує унікальну частину колекції Градобанку -
+            українське мистецтво другої половини 1980&nbsp;-1995 років.</p>
```

Comment:
Consider it will be better
```suggestion
          <p class="article__text">
            Національний Художній Музей України презентує унікальну частину колекції Градобанку -
            українське мистецтво другої половини 1980&nbsp;-1995 років.
          </p>
```